src/main/java/com/devonfw/ide/sonarqube/common/impl/check/DevonArchitectureComponentCheck.java

```diff
@@ -27,7 +27,6 @@ public void doScanFile(ClassTree tree, JavaFileScannerContext context) {
       this.configuration = new Configuration();
     }
     onConfigurationSet(context);
-    super.doScanFile(tree, context);
```

Comment:
IMHO we should either declare the topmost `doScanFile` method abstract or keep the super-call here (even if it currently does nothing, but that might change in the future and then it could cause problems).